src/Accordion/Accordion.tsx

```diff
@@ -5,9 +5,7 @@ type AccordionProps = React.HTMLAttributes<HTMLDivElement> & {
 };
 
 const Accordion = ({ accordion, ...rest }: AccordionProps): JSX.Element => {
```

Comment:
'accordion' is now an unused prop, so it needs to be removed here and from the type definition above.

Reply:
All done 👍